packages/messaging/content/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@botpress/messaging-content",
```

Comment:
Why not `@botpress/messaging-content-types` or `@botpress/messaging-types` ? If find messaging-content to be confusing.